task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -103,6 +104,12 @@ class TaskState(BaseModel):
     type: Literal["TaskState"] = "TaskState"
 
 
+class PatchTIToDeferred(TIDeferredStatePayload):
```

Comment:
```suggestion
class DeferTask(TIDeferredStatePayload):
```

might be a better name 🤷

Reply:
Yes that is better. Sounds like a command too. Where the task runner can call "defer this task for me"